### ISSUE TITLE
phpMyAdmin logs out if session folder has wrong permissions

### ISSUE THREAD

**Issue by liviuconcioiu** [phpmyadmin/phpmyadmin#16862]
### Describe the bug

I was logged on phpMyAdmin version when I updated PHP version to 8.0.5. It changed permissions from 777 to 770. When I refreshed the phpMyAdmin this error was given
```
phpMyAdmin - Error

Error during session start; please check your PHP and/or webserver log file and configure your PHP installation properly. Also ensure that cookies are enabled in your browser.

session_start(): open(SESSION_FILE, O_RDWR) failed: Permission denied (13)

session_start(): Failed to read session data: files (path: /var/lib/php/session)
```

After changing permissions back to 777, and refreshed the page, I was logged out. I don't know if it was supposed to do this or no.

### To Reproduce

1. Log in on phpMyAdmin
2. chmod session folder to 770
3. Refresh phpMyAdmin
4. A error is given because it can't write to session folder
5. chmod session folder to 777
6. Refresh phpMyAdmin
7. phpMyAdmin logs out

### Expected behavior

phpMyAdmin shouldn't log out, because the session file is still there, and is only a permissions problem.

### Server configuration

- Operating system: CentOS
- Web server: nginx/1.14.1
- Database version: MySQL 5.7.34
- PHP version: 8.0.5
- phpMyAdmin version: 5.0.4

### Client configuration

- Browser: Firefox
- Operating system: Windows

**Comment by williamdes** [comment]
> phpMyAdmin shouldn't log out, because the session file is still there, and is only a permissions problem.

I am not sure that this is an issue in our code base 🤔

**Comment by williamdes** [comment]
> phpMyAdmin shouldn't log out, because the session file is still there, and is only a permissions problem.

I would bet it does not logout, but because it is unable to read the session data, the session data is the void.
If you chmod back and <kbd>F5</kbd> does it work ?

**Comment by liviuconcioiu** [comment]
> 
> 
> > phpMyAdmin shouldn't log out, because the session file is still there, and is only a permissions problem.
> 
> I would bet it does not logout, but because it is unable to read the session data, the session data is the void.
> If you chmod back and F5 does it work ?

No, it doesn't work. The session file is still there, is not destroyed.

**Comment by williamdes** [comment]
Oh okay, so the session ID rotates
Not sure we can do anything about this PHP behaviour
What behaviour would you want?

**Comment by liviuconcioiu** [comment]
I think the best option is to check if the folder has read/write permissions first and don't attempt to read the session file directly like it does now, and show a message that permissions should be changed, before the page is refreshed.
